### DISCUSSION TITLE
Roadmap for v2.1

### DISCUSSION THREAD

**Discussion by semioticrobotic** [theopensourceway/guidebook#234] [General]
It's been a while, to be sure, but lately I've found myself making use of our guidebook and am wondering when all the changes we've merge since the v2.0 release will see the light of day.

@quaid: Is the project still active? Do you have plans to publish an updated version if the guidebook any time soon?

Happy to help where/as needed, if I can.

**Comment by quaid** [comment]
Hey @semioticrobotic, I know what you mean about using it all the time AND needing to energize for the 2.1 update.

Everyone it seems has been both busy and recovering from 2020 for a while, yet I wonder if you and I looked for one or two other folks to help us we could form up a little polish-and-release activity?

This is timely for me because I've been contemplating two new long chapters for the guide — which may arrive more as appendices than chapters, we'll see — and I'd appreciate warming up the engines on the ol' Millennium Falcon again to see if we can still make that Kessel run again (but this time, let's not go for the under 12 parsecs mark and just avoid that darn asteroid field, okay?!?)

**Comment by semioticrobotic** [comment]
Count me in, @quaid! Let me know how I can most be of service. I do love a good posse.

**Reply by quaid** [reply]
It seems the project kanban-style boards were deleted, at least I can no longer find the archived old style project boards — from fazed out to phasered out, it appears. We lost a bit of context on the v2.1 process because it was all laid out there. I thought there were labels for v2.1 as well and those don't seem to be in the repo either.

I went looking because I was going to say our next step was to scope the work to see what kind of help we needed. Now I think we need to find what new content is somewhere... Maybe I'm needlessly concerned? I need to get to my laptop and dig in a little more, let's see what we can find. Yeah

**Reply by quaid** [reply]
Ah-ha! It was just not visible on the mobile app, here's the kanban board in "classic" view:

https://github.com/theopensourceway/guidebook/projects/1

**Reply by quaid** [reply]
More importantly, I can see it was a milestone marker actually, and here is the 2.1 view:

https://github.com/theopensourceway/guidebook/projects/1?card_filter_query=milestone%3A2.1

These were certainly the ambitions. I do think I would expand on the community of practice content to include my full theory of communities and community management (#215). I've been working on the latter as a standalone blog post but it's extensive and would fit better in the full guide.

There are a few open issues/PRs to include, maybe we can get all these into a new-style project board for just this release?

Is there any other content you'd like to see included right now?

The two other topics I mentioned that might be appendices don't have sponsorship yet and it's on my near-term plan to work up an outline for each with a sample section or two as part of seeking sponsorship. I think those might be better to go on their own milestone like v2.1.1 or even v2.2.

Great question and topic, Bryan, and pretty good timing to boot!

**Comment by semioticrobotic** [comment]
> Here is the 2.1 view:
> 
> https://github.com/theopensourceway/guidebook/projects/1?card_filter_query=milestone%3A2.1
> 
> These were certainly the ambitions.

Agreed, and rather ambitious ambitions at that. What I'd support most enthusiastically is a more modest `v2.1` release that is mostly a bugfix and "update" release aimed at putting the revisions-to-date into production. 

Of course, if this is already well underway and already in something like a draft state, then including it in `v2.1` makes sense:

> I do think I would expand on the community of practice content to include my full theory of communities and community management (#215). I've been working on the latter as a standalone blog post but it's extensive and would fit better in the full guide. 

As for our editorial project board:

> There are a few open issues/PRs to include, maybe we can get all these into a new-style project board for just this release?

As arduous as it may be, I truly do think this is the most productive path forward: Remove all materials from the current ("classic-style") project board, delete that board, spin up a "new-style" project board that reflects [our editorial process](https://github.com/theopensourceway/guidebook/blob/main/EDITORIAL.md), and then identify the (fewer) issues we'd like to re-include in `v2.1`, putting them on the new board. This is likely a nice opportunity to rethink and reboot that little bit of infrastructure.

> Is there any other content you'd like to see included right now?

Nope! Just having the revisions and refactors already in the codebase pushed to the in-production guide would be very welcome!

**Comment by semioticrobotic** [comment]
Just an update that I cut a [2.1 release](https://github.com/theopensourceway/guidebook/releases/tag/v2.1) as part of the [guidebook refactor and migration to GitBook](https://github.com/theopensourceway/guidebook/discussions/237). We can close this discussion.
